src/pages/settings/custom-fields/payments/Payments.tsx

```diff
@@ -16,12 +17,15 @@ import { Field } from '../components';
 
 export function Payments() {
   const [t] = useTranslation();
+
   const title = `${t('custom_fields')}: ${t('payments')}`;
+
   const pages = [
     { name: t('settings'), href: '/settings' },
     { name: t('custom_fields'), href: '/settings/custom_fields' },
     { name: t('payments'), href: '/settings/custom_fields/payments' },
   ];
+
   useEffect(() => {
     document.title = `${import.meta.env.VITE_APP_TITLE}: ${t('custom_fields')}`;
   });
```

Comment:
Can we refactor this to use `useTitle` hook?

Reply:
Refactored!